### DISCUSSION TITLE
Quick join using PHP with joomla extension

### DISCUSSION THREAD

**Discussion by gitsimihub** [mynaparrot/plugNmeet-server#139] [Q&A] [answered]
I have an installation of PlugNmeet 1.1.1 as extension in Joomla 3.10 / PHP 7.4.32 and everything is working fine.
Now i would like to create a link to invite members.
The link should be valid for only one meeting.
How can i create such a link?

**Comment by jibon57** [comment, marked as answer]
At present one time usable link creation not possible with that extension but you can further development. At present you can create room & later can change password.
